src/Apps/Consign/Routes/MarketingLanding/Components/LandingPage/FooterBanner.tsx

```diff
@@ -1,19 +1,18 @@
-import { Box, Text } from "@artsy/palette"
+import { Box, Spacer } from "@artsy/palette"
 import { FullBleedBanner } from "Components/FullBleedBanner"
 import { RouterLink } from "System/Router/RouterLink"
 
 export const FooterBanner: React.FC = () => {
   return (
     <Box display={["none", "block"]}>
-      <FullBleedBanner dismissable={false} variant="defaultLight" mb={-4}>
-        <Text variant="xs">
-          Gallery or art dealer?{" "}
-          <RouterLink to="https://partners.artsy.net">
-            Become a partner
-          </RouterLink>{" "}
-          to access the world’s largest online marketplace.
-        </Text>
+      <FullBleedBanner dismissable={false} variant="defaultLight">
+        Gallery or art dealer?{" "}
+        <RouterLink to="https://partners.artsy.net">
+          Become a partner
+        </RouterLink>{" "}
+        to access the world’s largest online marketplace.
       </FullBleedBanner>
+      <Spacer y={-4} />
```

Comment:
Pardon, I misunderstood your comment. I thought about this and I think keeping it this way is the right thing to do but I might be wrong, I am interested to know what you think.
In order to not remove it, I will need to either inject a property within `LayoutDefault` [here](https://github.com/artsy/force/blob/77e1b4c396b58804aff86d07458388b670a9d6f5/src/Apps/Components/Layouts/LayoutDefault.tsx#L30) or create a new layout for it. Both options do not seem right to me because this used only here, what do you think?